qa/tasks/deepsea.py

```diff
@@ -660,10 +660,10 @@ def begin(self):
         self.sm.sync_pillar_data(quiet=self.quiet_salt)
 
     def end(self):
-        # self.log.debug("beginning of end method")
         super(DeepSea, self).end()
-        pass
-        # self.log.debug("end of end method")
+        self.log.info("in end(), ctx is {}".format(self.ctx))
```

Comment:
This is typical debug, not info.

Reply:
Indeed - I'm planning to drop this line as soon as I can test this PR. Right now, I don't even know if it will work.

Reply:
@smithfarm You can test it with storage-teuthology job on our internal jenkins.